source/_references/usermetadata.md

```diff
@@ -0,0 +1,113 @@
+---
+title: User metadata
+use: [references]
+---
+
+## Intro
+
+User metadata is metadata added to a source image by the API-user that doesn't change the image identifying hash.
+
+## User metadata types
+
+User metadata can be a string, integer, double, date, latitude/longitude or an array and also be changed after a image was uploaded.
+
+Default is string, if you want to store it as any other type, you have to specify it with a prefix
+
+| Type     | Prefix  |
+-----------|----------
+| Int      | int:    |
+| Double   | double: |
+| Date     | date:   |
+| Location | latlon: |
+| Array    | array:  |
+
+
+Date needs to be in the _YYYY-MM-DDThh:mm:ssZ_ format. Location needs to be a latitude and longitude in the following form, eg: "47.38,8.52".
+The elements of an array will always be stored as string, you can't specify a type there.
+
+## Add user metadata to a source image
+
+To add user metadata to a source image, you need to provide the organization, the identifying hash of the image, the name of the user metadata and the values to add.
```

Comment:
This sentence is of not much use IMHO as you see the API call afterwards. I'd replace it with nothing ;)